pootle/apps/pootle_store/forms.py

```diff
@@ -373,3 +407,129 @@ def save(self, **kwargs):
             super(UnitCommentForm, self).save(**kwargs)
 
     return UnitCommentForm
+
+
+class UnitSearchForm(forms.Form):
+
+    initial = forms.BooleanField(required=False)
+    count = forms.IntegerField(required=False)
+    path = forms.CharField(
+        max_length=2048,
+        required=True)
+    uids = MultipleArgsField(
+        field=forms.IntegerField(),
+        required=False)
+    filter = forms.ChoiceField(
+        required=False,
+        choices=UNIT_SEARCH_FILTER_CHOICES)
+    checks = forms.MultipleChoiceField(
+        required=False,
+        widget=CommaSeparatedCheckboxSelectMultiple,
+        choices=check_names.items())
+    category = CategoryChoiceField(
+        required=False,
+        choices=CATEGORY_CODES.items())
+    month = forms.DateField(
+        required=False,
+        input_formats=['%Y-%m'])
+    sort = forms.ChoiceField(
+        required=False,
+        choices=UNIT_SEARCH_SORT_CHOICES)
+
+    user = forms.ModelChoiceField(
+        queryset=get_user_model().objects.all(),
```

Comment:
I was thinking if we need here `default`, `nobody`. But seems they are necessary.